vortex-btrblocks/src/string.rs

```diff
@@ -195,9 +194,8 @@ impl Scheme for DictScheme {
         }
 
         // Find best compressor for codes and values separately
-        let downscaled_codes = downscale_integer_array(dict.codes().to_array())?.to_primitive()?;
         let compressed_codes = IntCompressor::compress(
-            &downscaled_codes,
+            &dict.codes().to_primitive()?,
```

Comment:
you removed downscaling here. Integer compressor only downscales in sparse and dict scheme scheme

Reply:
But then you moved it to the dict_encode function. Ignore me